core/base/supportedseries.go

```diff
@@ -82,12 +82,6 @@ func composeSeriesVersions() {
 	for k, v := range centosSeries {
 		allSeriesVersions[k] = v
 	}
-	allSeriesVersions[genericLinuxSeries] = seriesVersion{
-		WorkloadType: OtherWorkloadType,
-		OS:           genericLinuxOS,
-		Version:      genericLinuxVersion,
-		Supported:    true,
-	}
```

Comment:
Removal of generic linux means we don't support anything outside of ubuntu. Is this correct?